sql/migrations_test.go

```diff
@@ -15,5 +15,5 @@ func TestMigrationsAppliedOnce(t *testing.T) {
 		return true
 	})
 	require.NoError(t, err)
-	require.Equal(t, version, 9)
+	require.Equal(t, version, 10)
```

Comment:
Right now the test only fails if everything works correctly after adding a migration. If it passes without updating the assertion we wouldn't even notice that something is wrong 😅 

A more permanent fix would be:

```suggestion
	migrations, err := StateMigrations()
	require.NoError(t, err)

	expectedVersion := 0
	for _, migration := range migrations {
		expectedVersion = max(expectedVersion, migration.Order())
	}
	require.Equal(t, expectedVersion, version)
```